front/src/modules/ui/navbar/components/TaskNavMenuItem.tsx

```diff
@@ -0,0 +1,57 @@
+import { useNavigate } from 'react-router-dom';
+import { useRecoilState } from 'recoil';
+
+import { useTasks } from '@/activities/tasks/hooks/useTasks';
+import { NavItemProps } from '@/ui/navbar/components/NavItem';
+import {
+  StyledItem,
+  StyledItemCount,
+  StyledItemLabel,
+} from '@/ui/navbar/components/StyledNavItem';
+import { useIsMobile } from '@/ui/utilities/responsive/hooks/useIsMobile';
+
+import { isNavbarOpenedState } from '../../layout/states/isNavbarOpenedState';
+
+function NavItem({ label, Icon, to, onClick, active, danger }: NavItemProps) {
+  const navigate = useNavigate();
+  const [, setIsNavbarOpened] = useRecoilState(isNavbarOpenedState);
+
+  const { todayOrPreviousTasks } = useTasks();
+
+  const dueTasks = todayOrPreviousTasks?.filter(
```

Comment:
let's compute this dueTasks count in useTasks hook directly and inject it to NavItem in AppNavbar.tsx